azure-communication-ui/azure-communication-ui-demo-app/src/main/java/com/azure/android/communication/ui/callingcompositedemoapp/launcher/CallingCompositeJavaLauncher.java

```diff
@@ -27,7 +28,7 @@ public CallingCompositeJavaLauncher(final Callable<String> tokenRefresher) {
     }
 
     @Override
-    public void launch(final Context context,
+    public void launch(final MainActivity context,
```

Comment:
As above, don't mind renaming, but since I renamed MainActivity in another PR, I'd like to merge that first before renaming. If that can be merged then I'll bring this branch up to date and name it accordingly.

Reply:
nvm, will merge up (see the other is merged already) and update

Reply:
👍